Resources/Prototypes/DeltaV/Voice/speech_emotes.yml

```diff
@@ -2,242 +2,78 @@
 - type: emote
   id: HarpyHonk
   category: Vocal
-  chatMessages: [honks.]
+  chatMessages: [ honks ]
   chatTriggers:
-    - honk
-    - honk.
-    - honk!
-    - honk?
     - honks
-    - honks.
-    - honks!
-    - honks?
-    - honking
-    - honking.
-    - honking!
-    - honking?
-    - honked
-    - honked.
-    - honked!
-    - honked?
 
 - type: emote
   id: HarpyRing
   category: Vocal
-  chatMessages: [rings.]
+  chatMessages: [ rings ]
   chatTriggers:
-    - ring
-    - ring.
-    - ring!
-    - ring?
     - rings
-    - rings.
-    - rings!
-    - rings?
-    - ringing
-    - ringing.
-    - ringing!
-    - ringing?
-    - ringed
-    - ringed.
-    - ringed!
-    - ringed?
-    - rang
-    - rang.
-    - rang!
-    - rang?
```

Comment:
Here too, and in all other places. Don't remove the different tenses of words. It's only necessary to remove punctuation variants.